app/src/main/java/de/fraunhofer/fokus/OpenMobileNetworkToolkit/Iperf3/Iperf3Parser.java

```diff
@@ -15,20 +17,35 @@
 
 
 public class Iperf3Parser {
+    private static final String TAG = Iperf3Parser.class.getSimpleName();
+
+    public static final Object END_MARKER = new Object();
```

Comment:
get rid of this.

Reply:
or you know what, just let it sit for a while, and when it causes conflicts then delete it. it's just used as a marker value to fire the "end" property change so we see when the parsing is over.